tmt/utils.py

```diff
@@ -747,6 +749,34 @@ def environment_to_dict(variables):
     return result
 
 
+def environment_file_to_dict(env_files: Iterable[str]) -> Dict[str, str]:
+    """Create dict from files.
+
+    Files should be in yaml/yml or dotenv format.
+
+    dotenv file example:
+        ```bash
+        A=B
+        C=D
+        ```
+    yaml file example:
+        ```yaml
+        A: B
+        C: D
+        ```
+    """
+    res = {}
+    for env_file in map(Path, env_files):
+        assert env_file.is_file(), f"{env_file} doesn't exist"
```

Comment:
Should raise exception (tmt.utils.GeneralError) which is processed nicely to the user.